package.json

```diff
@@ -1,32 +1,35 @@
 {
-  "name": "puchacz",
-  "version": "0.1.0",
-  "private": true,
-  "scripts": {
-    "dev": "next dev",
-    "build": "next build",
-    "start": "next start",
-    "lint": "next lint"
-  },
-  "dependencies": {
-    "@next/font": "^13.0.7",
-    "@types/node": "18.11.16",
-    "@types/react": "18.0.26",
-    "@types/react-dom": "18.0.9",
-    "clsx": "^1.2.1",
-    "eslint": "8.30.0",
-    "eslint-config-next": "13.0.7",
-    "next": "13.0.7",
-    "react": "18.2.0",
-    "react-dom": "18.2.0",
-    "typescript": "4.9.4"
-  },
-  "devDependencies": {
-    "@svgr/webpack": "^6.5.1",
-    "autoprefixer": "^10.4.13",
-    "postcss": "^8.4.20",
-    "prettier": "^2.8.1",
-    "prettier-plugin-tailwindcss": "^0.2.1",
-    "tailwindcss": "^3.2.4"
-  }
+	"name": "puchacz",
+	"version": "0.1.0",
+	"private": true,
+	"scripts": {
+		"dev": "next dev",
+		"build": "next build",
+		"start": "next start",
+		"lint": "next lint",
+		"cy:open": "cypress open",
+		"cy:dashboard": "cypress run --record --key 596678da-39f8-47da-addd-104baa5fd9c3"
```

Comment:
We don't need that cy:dashboard since we are not going to record local tests, and currently we are showing our secret key. Please delete that one and generate a new one.

Reply:
New key generated, deleteded cy:dashboard